---
layout: page
title: "Academic Curriculum Vitae"
permalink: /academic/
---

Background & Specialty
======================

Since 2007, my research involve risk prediction for cancer, cell aging,
immune and brain disorders across France, Canada, and the United States.
Doctor and consultant in the fundamentals of genetics, functional
genomics, and bioinformatics. Specialization in model & non-model
species (14 years), sequencing pipelines (1050 hours), and machine
learning (770 hours) using R (7300 hours), Bash (5720 hours), and Python
(200 hours). Research interests cover pipeline automation, systems
biology, gene networks, clustering, machine learning, warehousing data
and performance computing (PBS, MySQL, Spark), single cell sequencing,
and heterogeneous data integration ([public
pipelines](https://github.com/neocruiser/pipelines), [peer-reviewed
published projects](https://github.com/neocruiser/Rstats) &
[website](https://neocruiser.github.io)).

Education
=========

Ph.D., Computational & Functional Genomics, [**University of
Quebec**](http://www.uqar.ca/), Canada *Summer 2014*

M.Sc., Genetics & Molecular Biology, [**University of Henri
Poincaré**](http://formations.univ-lorraine.fr/fr-FR/liste), Nancy,
France *Winter 2009*

B.Sc., Behavioural Genetics & Parasitology, **University of Lebanon**,
Lebanon *2005 - 2006*

Premedical studies, **University of Lebanon**, Lebanon *2004*

Professional Employment
=======================

Senior computational biologist, [**Princess Margaret Cancer
Centre**](http://www.uhn.ca/PrincessMargaret/), Toronto, Canada *2017 -
2018*

Postdoctoral associate, [**University of Stony
Brook**](http://www.stonybrook.edu/), Stony Brook, United States *2015 -
2017*

Bioinformatics research scientist, [**University of
Vermont**](http://www.uvm.edu/), Vermont, United States *Summer 2014*

Assistant researcher, [**University of Western
Brittany**](https://www.univ-brest.fr/), Brest, France *2011 - 2013*

Assistant researcher, [**University of Quebec at
Rimouski**](http://www.uqar.ca/), Quebec, Canada *2009 - 2014*

Research associate, [**University of
Lorraine**](http://www.univ-lorraine.fr/), Nancy, France *2008 - 2009*

Research scientist, [**University of Montpellier
2**](http://www.english.univ-montp2.fr/), Montpellier, France *Summer
2007*

Research scientist, [**University of
Lorraine**](http://www.univ-lorraine.fr/), Nancy, France *Winter 2007*

Lab assistant, **Human Pathology Lab, Medlab**, Lebanon *Winter 2005*

Grants & Awards
===============

Award for Deep Learning & Reinforcement Learning Summer Schools in
Toronto, funded by the Canadian Statistical Sciences Institute ,
**\$1000** *2018*

**PI** award in the US. Unraveling molecular processes associated with
shellfish resistance to environmental and pathological stress.
University of Stony Brook, funded by the at Pittsburgh Supercomputing
Center USA NSF, **\$61,209** *2016-2017*

**PI** award in the US. Investigating the health of marine animals and
mechanisms of diseases caused by pathogenic microbes. University of
Stony Brook, funded by the USA, **\$20,340** *2016-2017*

Fellowship in the US. National Sea Grant College Program at the Research
Foundation of State University of New York. Funded by and by the ,
**\$94,565** *2015-2017*

**Co-PI** award. Characterizing the effects of environmental challenges
on genome dynamics of marine parasites. University of Stony Brook,
funded by NSF, **\$3390** *2015*

Travel award in Paris. Statistical workshop in linear prediction and
non-linear analyses of categorical genetic traits, funded by [University
of Quebec](https://www.uqar.ca/services/services-a-l-etudiant/bourses),
**\$3,000** *2012*

Cumulative travel awards in Canada. Réseau international francophone en
écotoxicologie aquatique for analyses of micropollutants, funded by
[University of
Quebec](https://www.uqar.ca/services/services-a-l-etudiant/bourses),
**\$5,000** *2010-2012*

National award in Canada. New genomic approaches to discover growth and
mortality biomarkers, funded by , **\$75,000** *2009-2014*

Peer reviewed Publications
==========================

**Bassim, S.**, Allam, B. SNP hot-spots in the clam parasite QPX. *BMC
Genomics*, 19:486, 2018. <span>****</span>

**Bassim, S.**, Chapman, W. R., Tanguy, A., Moraga, D., Tremblay, R.
Predicting growth and mortality of bivalve larvae using gene expression
and supervised machine learning. *Comparative Biochemistry and
Physiology - Part D: Genomics and Proteomics*. 16, 2015, 59-72.\
<span>****</span>

**Bassim, S.**, Tanguy, A., Génard, B., Moraga, D., Tremblay, R.
Identification of *Mytilus edulis* genetic regulators during early
development. *Gene*, 555(1), 65–78, 2014.\
<span>****</span>

**Bassim, S.**, Genard, B., Gauthier-Clerc, S., Moraga, D., Tremblay, R.
Ontogeny of bivalve immunity: assessing the potential of next-generation
sequencing techniques. *Reviews in Aquaculture*, 6, 1–21, 2014.
<span>****</span>

**Bassim, S.**, Allam, B. Network analysis of dual RNA-seq reveals
conserved virulence mechanisms of the protistan parasite QPX. *BMC
Genomics*.

**Bassim, S.**, Allam, B. Host-parasite interactions in non-model
species: From RNA-seq assembly to network inference using machine
learning and high-performance computing *BioRXiv*.

**Bassim, S.**, Kridel, R. Transcriptional polygenic score for diffuse
large B cell lymphoma identify individuals with risk of central nervous
system relapse *BioRXiv*.

**Bassim, S.**, Allam, B. Espinosa, E. Cross-tissue network analysis of
*Crassostrea virginica* neuropeptide gene expression and their
association to gill functions. *eLife*.

Invited Talks
=============

Bassim, S. *Invited talk*: Detection of high-risk lymphoma patients
using machine learning at the [Vector](https://vectorinstitute.ai/)
institute for artificial intelligence, Toronto. July 2018.

Bassim, S. Scott, D., Kridel R., *Poster*: Determinants of central
nervous system relapse in diffuse large B-cell lymphoma, Accelerating
precision medicine, [UHN](http://www.uhnresearch.ca/), Toronto, Canada.
January 2018.

Bassim, S., Allam, B. A transcriptomic approach to characterize
host-parasite interactions, National Shellfisheries Association (NSA),
*Invited speaker* in Comparative Genomics, Tennessee, March 2017.

Bassim, S. Introduction to computational genomics *Invited lecturer*,
Stony Brook University. New York, June 2015.

Bassim, S. Genomic and physiological processes of the blue mussel
*Mytilus edulis* (L., 1758) during early development and the impact of
eicosanoid precursors. *Invited speaker & Poster*: *ECOBim,
International workshop on environmental assessment*. Montreal, Canada.
May 20–24, 2013.

Bassim, S. Transcriptomic analyses and physiological effectors of
*Mytilus edulis* during larval ontogeny. *Invited speaker & Poster*:
*ECOBim, International workshop on environmental assessment*. Reims,
France, May 10–15, 2012.

Bassim, S., Murad, H., Devignes, MD., Ndiaye, B., Becuwe, P., Domenjoud,
L. Discovery of new PPAR direct target genes in the glioblastoma cell
line T98G. *Poster*: *Communication: Conference of the Grand-East region
in oncology research.* Faculty of sciences, Nancy 1, France. 2009c.
Poster abstract.

Bassim, S., Devignes, MD., Ndiaye, B., Becuwe, P., Domenjoud, L.
Successful in silico application of immunoprecipitated chromatine (ChIP)
for the analysis of the promoter region of PPAR target genes. Master
Thesis, University of Lorraine Nancy 1, Faculty of Sciences, France,
2008.

Bassim, S., Guédon, G., Study of the accretion and mobilization of
integrative and conjugative elements (ICEs) and their genomic variants
in *Streptococcus thermophilus*. Graduate Thesis, University of Lorraine
Nancy 1, Faculty of Sciences, France, 2007.

Books & Reports
===============

Bassim, S. Genomic and physiological processes during early development
of the blue mussel *Mytilus edulis* (L., 1758): Impact of eicosanoid
precursors. *Ph.D Dissertation, University of Quebec at Rimouski,
Canada*, 2014.

Bassim, S. Molecular and physiological mechanisms associated with
increased growth and low mortality rates during early larval development
of the blue mussel *Mytilus edulis* (L., 1758). *Seminar, Institute of
Sciences and Oceans, Quebec, Canada*, 2012.

Bassim, S. Consequences of the Baltic invasion of the Ctenophora species
*Mnemiopsis leidyi* on the different trophic networks. *Report, Doctoral
program, University of Quebec at Rimouski, Canada*, 2009b.

Bassim, S. Consequences of a lunar anomaly on the Saint-Lawrence Golf
and estuary. *Report, Doctoral program, University of Quebec at
Rimouski, Canada*, 2009a.

Bassim, S., Boumenir, Y. Characterisation of the intraspecies
polymorphism of *Streptococcus thermophilus*. *Report, Laboratory of
genetic and microbiology, University of Lorraine Nancy 1, Faculty of
Sciences, France*, 2007.

Teaching & Supervision
======================

Workshop , RNA-seq & bioinformatics (5 days, &lt;5 members, University
of New York) *Summer 2017*

Computer science, statistics & biology at the [Nassau County Science
Competition (NCSC)](https://www.cstl.org/), New York, USA *June 2017*

Computer science & molecular biology at the [WAC Lighting Foundation
Invitational Science Fair](http://www.researchassociation.org/), New
York, USA *April 2017*

Computational biology & bioinformatics at the [Science & Engineering
Fair (LISEF)](http://www.lisef.org/?content_code=home), New York, USA
*March 2017*

Science at the [Bay Scallop
Bowl](http://you.stonybrook.edu/bayscallopbowl), New York, USA *February
2017*

Computational biology & bioinformatics at the [Science & Engineering
Fair (LISEF)](http://www.lisef.org/?content_code=home), New York, USA
*February 2017*

Workshop , data mining in transcriptomics (5 hours, &lt;20 members,
University of New York) *Fall 2016*

Supervision in bioinformatics for Taejin Park (postdoc, University of
New York) *Summer 2016*

Computer science & molecular biology at the [WAC Lighting Foundation
Invitational Science Fair](http://www.researchassociation.org/), New
York, USA *March 2016*

Supervision in bioinformatics for Amelie Segara (postdoc, University of
New York) *Summer 2015*

Supervision in statistics for Guangchen Liu (PhD, University of Vermont)
*Winter 2015*

LaTeX for Anne-Sophie Sainte Marie (MSc, University of Quebec) *2013*

Supervision in genomics for Camille Lacroix (PhD, University of Western
Brittany) *2012*

Supervision in molecular biology & genomics for Judith Rumin (MSc,
University of Western Brittany) *2012*

Supervision in cell biology for Fernando Silva (PhD, University of
Antofagasta) *2011*

Java for Yasmine Boumenir (MSc, University of Nancy) *Winter 2008*

Interpreted languages & Bioinformatics
======================================

(12 years)

R/Bioconductor (9 years), Bash (7 years), Perl (2 years), Python (2
year), emacs-lisp (2 years)

Core omics tools, pipeline automation, data mining & databases (8 years
sequence analysis, clustering, variant calling, annotation, expression &
others)

High performance computing Torque PBS (4 years, XSEDE,
[Bridges](http://www.psc.edu/index.php/bridges-approach),
[Greenfield](https://psc.edu/index.php/resources-for-users/computing-resources/greenfield),
[Comet](https://portal.xsede.org/sdsc-comet),
[LI-red](http://www.iacs.stonybrook.edu/resources/handy-accounts#overlay-context=resources/accounts),
Amazon AWS)

MySQL (1 year), Hadoop

[Tensorflow](https://www.tensorflow.org/) and Keras (&lt;1 year)

[Theano/Lasagne](https://github.com/Lasagne/Lasagne) & [H2O/Sparkling
Water](http://www.h2o.ai/) (&lt;1 year)

(10 years)

Machine Learning (neural networks, kernelized support vectors, decision
trees, dimension reduction, clustering, bagging, boosting, resampling,
mini-batch stochastic sampling, regularization, unsupervised learning)

Ordination
([PCA](https://en.wikipedia.org/wiki/Principal_component_analysis),
[CA](https://en.wikipedia.org/wiki/Correspondence_analysis),
[MCA](https://en.wikipedia.org/wiki/Multiple_correspondence_analysis),
Hierarchical MFA,
[FAMD](https://en.wikipedia.org/wiki/Factor_analysis_of_mixed_data)) and
Bayesian modeling

(10 years)

R, [Power BI](https://powerbi.microsoft.com/en-us/), Circos, and
[D3.js](http://d3js.org/) (HTML, CSS, JavaScript), Networks
([Cytoscape](http://cytoscape.org/what_is_cytoscape.html),
[Gephi](https://gephi.org/features/))

Emacs, Visual Studio, Org, Illustrator, InDesign, Blender, Markdown,
GitHub

TeX (LaTeX, <span><span>B-.05em<span>i-.025em b</span>-.08em
TeX</span></span>), KnitR, and Beamer

Projects & Experience
=====================

**Machine learning & risk prediction in Lymphoma** (1037 hours) *Winter
2018*\
Machine and deep learning pipeline to classify patients with Lymphoma
into low or high risk candidates based on clinical data. Classifier
back-end on genetic networks, regularization, and mutation data from
exome sequencing of 5 patients. [Github
link](https://github.com/neocruiser/Rstats/tree/master/relapse)

**Host and parasite cross-talk and infection mechanisms** (846 hours)
*Spring 2017*\
Constructed a pipeline to separate and de novo assemble dual RNA-seq
pathogen and host transcriptomes, classify and cluster functions.
Inferred cross-species interaction networks. [Github
link](https://github.com/neocruiser/Rstats/tree/master/nodule)

**Genetic interactions that link neuropeptides to feeding behavior**
(1853 hours) *Summer 2016*\
Constructed pipeline for de novo assembly and regulatory network
construction from non-model RNA-seq data. Inferred gene-gene
interactions, pathway clustering, annotation, and machine learning
comparison of cell lines with neural networks. [Github
link](https://github.com/neocruiser/Rstats/tree/master/ganglia)

**Shotgun metagenomic analyses** (56 hours) *Spring 2016*\
Constructed a classification pipeline to phylogenetically associate DNA
shotgun bacteria and characterized pathogenic interactions with host
from metagenomic sequenced samples derived from oyster mucus. [Github
link](https://github.com/neocruiser/Rstats/tree/master/metageno16)

**Nature and origin of parasite in RNA-seq host/pathogen samples** (226
hours) *Winter 2016*\
Constructed pipeline for de novo assembly and parasite classification
from dual RNA-seq infected host tissues for comparative genomics
analysis.

**Viral genome assemblies in high throughput sequencing samples** (100
hours) *Winter 2016*\
Constructed pipeline to assemble DNA shotgun virus genomes from marine
animal samples for pathogen classification (virus, bacteria, archae,
plasmids) and their affecting abundance on host protein synthesis.

**Structured database for genetic interrogation and discovery** (20
hours) *Summer 2015*\
Constructed a MySQL database that includes genetic variants, RNA-seq
gene expression, annotations from 20 public libraries, gene-gene
interactions, ontologies, and metabolic pathways.

**New genes for parasite virulence and aggressiveness** (625 hours)
*Summer 2015*\
Developed technique to call genetic mutations in different parasitic
strains without annotated genome. Genomic hotspots identified from
RNA-seq data. [Github
link](https://github.com/neocruiser/Rstats/tree/master/rnaseQPX)

**Supervised prediction of Parkinson’s disease based on clinical data**
(307 hours) *Summer 2014-Fall 2015*\
Designed new back-end pipeline built on a machine learning model with
[elastic net](https://en.wikipedia.org/wiki/Elastic_net_regularization)
and [SVM](https://en.wikipedia.org/wiki/Support_vector_machine) to
classify and correlate mutations with common psychiatric disorders
within different world populations. [Github
link](https://github.com/neocruiser/Rstats/blob/master/brain.org#2-plink)

**Machine learning predicting cellular growth and mortality** (605
hours) *2012-14*\
Developed new machine learning ensemble model with
[lasso](https://en.wikipedia.org/wiki/Least_squares#Lasso_method),
[ANN](https://en.wikipedia.org/wiki/Artificial_neural_network),
[boosted](https://en.wikipedia.org/wiki/Gradient_boosting)-[GLM](https://en.wikipedia.org/wiki/Generalized_linear_model),
and [bagging](https://en.wikipedia.org/wiki/Bootstrap_aggregating) for
classification of cellular mechanisms. Genes responsible in growth and
mortality were characterized from RNA-seq and microarray timeseries
data. [Github
link](https://github.com/neocruiser/thesis2014/blob/master/Paper3/paper3.R)

**New transcription factors that regulate growth and metamorphosis**
(1020 hours) *2010-11*\
Developed RNA-seq and microarray pipelines. First time implementation on
marine [invertebrate
larvae](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=erctameenrartcd&acc=GSE55580)
of Bayesian model to infer gene-gene interactions. Identified gene
networks that trigger metamorphosis, lower immune defense, and increase
metabolism. [Github
link](https://github.com/neocruiser/thesis2014/blob/master/ebdbn/ebdbn.R)

**Role of
[PPARs](https://en.wikipedia.org/wiki/Peroxisome_proliferator-activated_receptor)
in glioblastoma and breast cancer** *Fall 2009*\
Identified PPAR-regulated genes. Protein-DNA (ChIP) essays validated the
genes cell migration function & their potential in cancer therapy.

**Designing analyses that estimate mutation affinity** *Fall 2008*\
Developed a pattern recognition technique in collaboration with
[Laboratoire Lorrain de Recherche en Informatique et Ses
Applications](http://hal.inria.fr/LORIA/en/), (PI: [Dr. Marie-Dominique
Devignes](http://www.researchgate.net/profile/Marie-Dominique_Devignes)).
Recognized 7 cancer-causing genes
([PPARs](https://en.wikipedia.org/wiki/Peroxisome_proliferator-activated_receptor))
& their high-affinity binding sites in the human genome.

**Lowering rheumatoid arthritis severity in mouse** *Spring 2008*\
Constructed
[vectors](https://en.wikipedia.org/wiki/Vector_(molecular_biology)) with
[microRNA](https://en.wikipedia.org/wiki/MicroRNA). Carried targeted
gene silencing of specific interleukins in *E. coli* and macrophages.

**In vitro bacteria transformation** *Spring 2007*\
Constructed vectors to transform bacteria with antibiotic resistance
genes. Validated in vitro. Optimizing bacteria conjugation.

**Identify genes for transfer of antibiotic resistance** *Fall 2007*\
Constructed bacterial
[vectors](https://en.wikipedia.org/wiki/Vector_(molecular_biology)).
Isolated, extracted, and targeted genes that
[transpose](https://en.wikipedia.org/wiki/Transposable_element) during
bacterial conjugation (pre-CRISPR).

Meetings & Workshops
====================

Deep & Reinforcement Learning summer school hosted by the Canadian
Institute For Advanced Research ([CIFAR](https://www.cifar.ca/)) &
[Vector](https://vectorinstitute.ai/). Special invitation. Participation
by Google Brain, DeepMind, Microsoft research, Alberta Machine
Intelligence Institute & Institut québécois d’intelligence artificielle
(MILA). Supported by Drs. [Yoshua
Bengio](http://www.iro.umontreal.ca/~bengioy/yoshua_en/index.html) &
[Richard Sutton](http://incompleteideas.net/), Toronto, (10 days) July
2018

Meeting for machine learning & genomics, [CIFAR](https://www.cifar.ca/)
& [GETx](http://www.getconference.org/) Brave new world: Genetic
networks to decode complex disease, Toronto, June 2018

Meeting for transcriptomics and cancer research, Terry Fox Research
Institute symposium. Deconstructing tumor complexity, Toronto, December
2017

Xsede workshop on Hadoop and Spark with Bridges, March 2016

Grant winning grant proposals, October 2015 (Dr. [John D.
Robertson](http://www.grantcentral.com/about-us/professional-staff/john-d-robertson/))

Workshop on Machine Learning, Stanford University, USA, February –
April, 2014 (Drs. [Trevor Hastie](https://web.stanford.edu/~hastie/) &
[Robert Tibshirani](http://statweb.stanford.edu/~tibs/)).

Workshop on Intellectual property and Patents, July 2013 ([Dr. Louise G.
Bernier](https://www.robic.ca/en/2016/01/first-patent-program-great-start-protecting-innovation/)).

ECOBim, Ecotoxicology in marine ecosystems, Montreal, Quebec, Canada,
May 2013.

Workshop on Bayesian Networks, June, Paris, France, 2012 ([Dr. Daniel
Gianola](https://en.wikipedia.org/wiki/Daniel_Gianola)).

ECOBim, Ecotoxicology in marine ecosystems, Reims, France, May 2012.

ECOBim, Ecotoxicology in marine ecosystems, Saguenay, Quebec, Canada,
May 2010.
